doc/developer/tooling.rst

```diff
@@ -46,10 +46,10 @@ Don't open until it is ready
 
 Your PR isn't complete when the code works, it is complete when the code is polished and all the
 tests are written and working. The idea here is: as soon as you open a PR, people will start
-spending their time looking at it. And their time is valuable. An exception to this rule is that
-GitHub allows you to `open a Draft PR <https://github.blog/2019-02-14-introducing-draft-pull-requests/>`_
-which is a nice option if you need to open your PR early for some reason (usually testing). You
-can also convert any open PR to Draft if you decide it needs more work.
+spending their time looking at it. And their time is valuable. Even though GitHub allows you to
+`open a Draft PR <https://github.blog/2019-02-14-introducing-draft-pull-requests/>`_, this is
+not the default open in ARMI. It should not be your workflow to open a Draft PR by default. We
+prefer to keep the PR list as short as possible.
```

Comment:
Not sure about this chunk... Did this come down as a directive from somewhere?

Reply:
Honestly?

People are abusing "Draft PRs" in ARMI.

People open a Draft PR into ARMI before the work is even CLOSE to do, and they leave it open for a month, making commit after commit.

Which means every commit someone makes to an open ARMI PR, and they fail CI because of `black` or something, I get an email. Or 8 emails. 

If the work is not done, do not open a PR.  You are wasting my time.  If a PR is open for a month, be sure that I waste my time looking at it at least 5 times a day, every day (including weekends) for the entire month.

Reply:
I understand there are internal projects at TerraPower that use Draft PRs differently.  That's fine for them.  But those rules and processes do not apply to ARMI.

Reply:
> If a PR is open for a month, be sure that I waste my time looking at it at least 5 times a day, every day (including weekends) for the entire month.

I get it when some PRs get open prematurely and then just hang. Shoot, my PR #1386 falls victim to this. But sometimes A PR could are open for review and then get held up for other downstream dependencies. Other downstream PRs, unexpected issues with testing, schedule shifts, etc. Should those get closed in that case? One of my PRs falls into this category (PR #1376). That work is done, but there are other downstream dependencies blocking it from being open and ready to merge.

I was under the impression that leaving PRs as draft was fine and that only PRs that are "ready to go" should be open. But maybe that's not the case. What's your guidance/preference here @john-science ?

Reply:
If a PR is opened and then found to be faulty, it can be temporarily converted to Draft. Sure.

But that's about it.

This PR: https://github.com/terrapower/armi/pull/1338 has been open for three months.  I have looked at it hundreds of times.  Is it abandoned? Why are there still merge conflicts? I don't know. And before the merge conflict this one PR caused me to get nearly 200 emails.

Reply:
I have removed this and am moving it to another PR.